compress/decompress.go

```diff
@@ -372,9 +378,9 @@ func (g *Getter) HasNext() bool {
 // After extracting next word, it moves to the beginning of the next one
 func (g *Getter) Next(buf []byte) ([]byte, uint64) {
 	savePos := g.dataP
-	l := g.nextPos(true)
-	l-- // because when create huffman tree we do ++ , because 0 is terminator
-	if l == 0 {
+	nextPos := g.nextPos(true)
```

Comment:
are you sure this is nextPos and not lenght of word?

Reply:
not sure, but nextPos looks cleaner than some `l` defined at the function's beginning. could rename to `wordLength` or `patternLength`

Reply:
yes, please rename - to something that shows real meaning of variable (I also don't know - what it means :-) )